### PR TITLE
[#66] update zgw-consumers

### PR DESCRIPTION
**Pull request by SonnyBA**
Partially fixes https://github.com/maykinmedia/open-api-framework/issues/66

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1 +1,3 @@
 open-api-framework
+
+git+https://github.com/maykinmedia/commonground-api-common@feature/update-notifs-client
```
Comment (stevenbal) [inline requirements/base.in:3]: Once this is released, we should bump it in open-api-framework and then bump that instead. Open Klant doesn't seem to be affected by the zgw-consumers update, so let's keep this PR open and change it once the releases are done

Comment (SonnyBA) [reply requirements/base.in:3]: This also applies to https://github.com/open-zaak/open-zaak/pull/1800 and https://github.com/open-zaak/open-notificaties/pull/196 and the upcoming PR's for the other projects too.

**Review by annashamray — APPROVED** [review]
Please update it and merge after the new `zgw_consumers` version is released

File: requirements/base.txt [hunk 2]
```diff
@@ -105,6 +111,7 @@ django==4.2.15
     #   notifications-api-common
     #   open-api-framework
     #   zgw-consumers
+    #   zgw-consumers-oas
```
Comment (annashamray) [inline requirements/base.txt:114]: wait why do we need this dep? I don't remember this being part of commonground-api-common

**Comment by SonnyBA** [comment]
This requires a new `commground-api-common` release with https://github.com/maykinmedia/commonground-api-common/pull/45

File: requirements/base.in [hunk 3]
```diff
@@ -1 +1,3 @@
 open-api-framework
+
+commonground-api-common>=2.0.1
```
Comment (stevenbal) [inline requirements/base.in:3]: this can be upgraded to 2.1.0

**Review by stevenbal — APPROVED** [review]
Just the comment about bumping commonground-api-common, looks good otherwise

**Comment by SonnyBA** [comment]
Requires https://github.com/maykinmedia/commonground-api-common/pull/50

**Pull request merged**